eclass/qmake-utils.eclass

```diff
@@ -116,4 +116,96 @@ eqmake5() {
 	fi
 }
 
+# @FUNCTION: qt6_get_bindir
```

Comment:
Please rebase. We already had the `qt6_get_*` functions in the qt overlay, looks like they've just been added to gentoo.git as well.

And please note commit 844eff16165fe6b551f6dc5ba3f7c4cb6cc21cc3 adding the RELEASE_WITH_DEBUGINFO flags.

Reply:
great it's already in gentoo.git. rebased as propsed... :+1: